back-end/src/main/kotlin/Gneiss/PacketCompiler/Service/Packet.kt

```diff
@@ -1,13 +1,37 @@
 package Gneiss.PacketCompiler.Service
 
 class PacketPostRequest(val invoicePDFPath: String, val approvalPDFPath: String, val csvPDFPath: String, val compiledPDFPath: String)
+class PacketPatchRequest(val invoicePDFPath: String?, val approvalPDFPath: String?, val csvPDFPath: String?, val compiledPDFPath: String?)
 
 class PacketPostResponse()
+class PacketPatchResponse()
 
 class Packet {
     companion object {
+
+        var setInvoicePDFPath: String? = null
```

Comment:
This Packet object won't represent a single packet, preferably each function here won't be manipulating any state. I would just get rid of these variables.

Reply:
Actually the fact that the `Packet` class _doesn't_ represent a single packet probably means we've chosen a bad name for the class :joy:. Maybe it should be `PacketRequestHandler`? Maybe something to address next sprint.